src/display/api.js

```diff
@@ -178,6 +178,13 @@ PDFJS.openExternalLinksInNewWindow = (
     false : PDFJS.openExternalLinksInNewWindow);
 
 /**
+ * Unique identifier for each PDF document loaded in a single page
+ * @var {number}
+ */
+ PDFJS.documentID = (PDFJS.documentID === undefined ?
```

Comment:
Why would we need to expose this from the API? It seems like something we would want to keep internal.